src/app/detail/[task_id]/[user_type]/actions.ts

```diff
@@ -121,3 +121,10 @@ export const postScrapImage = async (token: string, payload: ScrapImageRequest)
     body: JSON.stringify(payload),
   })
 }
+
+async function getSignedUrlUpload(token: string, fileName: string) {
+  const res = await fetch(`${apiUrl}/api/attachments/upload?token=${token}&fileName=${fileName}`)
+
+  const data = await res.json()
+  return data.signedUrl
+}
```

Comment:
Server actions should only be used for mutations, since they expose a POST endpoint not a GET endpoint. It seems bad to "get" data from a POST endpoint